src/components/remote_control/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -0,0 +1,411 @@
+    "{\"jsonrpc\":\"2.0\",\"id\":51,\"result\":{\"code\":21,\"method\":\"RC."
+    "SetInteriorVehicleData\",\"moduleData\":{\"moduleType\":\"CLIMATE\","
+    "\"ControlData\":{\"Speed\":0,\"outsideTemperature\":{\"unit\":"
+    "\"CELSIUS\",\"value\":\"high\"},\"desiredTemperature\":{\"unit\":"
+    "\"CELSIUS\","
+    "\"value\":25},\"acEnable\":true,\"circulateAirEnable\":true,"
+    "\"autoModeEnable\":true,\"defrostZone\":\"ALL\",\"dualModeEnable\":true,"
+    "\"acMaxEnable\":true,\"ventilationMode\":\"BOTH\"}},\"isSubscribed\":"
+    "false}}";
+}
+
+namespace test {
+namespace components {
+namespace remote_control_test {
+namespace set_interior_vehicle_data_request_test {
+
+class SetInteriorVehicleDataRequestTest : public ::testing::Test {
+ public:
+  typedef utils::SharedPtr<remote_control::commands::BaseCommandRequest>
+      SIVDRequestPtr;
+  typedef rc_event_engine::Event<application_manager::MessagePtr, std::string>
+      SIVD_HMI_Response;
+  SetInteriorVehicleDataRequestTest()
+      : mock_service_(utils::MakeShared<NiceMock<MockService> >())
+      , mock_app_(utils::MakeShared<NiceMock<MockApplication> >())
+      , rc_app_extention_(
+            utils::MakeShared<remote_control::RCAppExtension>(kModuleId)) {
+    ON_CALL(*mock_app_, app_id()).WillByDefault(Return(app_id_));
+    ON_CALL(mock_module_, resource_allocation_manager())
+        .WillByDefault(ReturnRef(mock_allocation_manager_));
+    ON_CALL(mock_module_, service()).WillByDefault(Return(mock_service_));
+    ON_CALL(*mock_service_, GetApplication(app_id_))
+        .WillByDefault(Return(mock_app_));
+    EXPECT_CALL(mock_module_, event_dispatcher())
+        .WillRepeatedly(ReturnRef(event_dispatcher_));
+    ServicePtr exp_service(mock_service_);
+    mock_module_.set_service(exp_service);
+  }
+
+  remote_control::request_controller::MobileRequestPtr CreateCommand(
+      application_manager::MessagePtr msg) {
+    return remote_control::RCCommandFactory::CreateCommand(msg, mock_module_);
+  }
+
+  application_manager::MessagePtr CreateBasicMessage() {
+    application_manager::MessagePtr message = utils::MakeShared<Message>(
+        MessagePriority::FromServiceType(protocol_handler::ServiceType::kRpc));
+    message->set_connection_key(kConnectionKey);
+    message->set_function_id(RCFunctionID::SET_INTERIOR_VEHICLE_DATA);
+    message->set_function_name("SetInteriorVehicleData");
+    return message;
+  }
+
+ protected:
+  utils::SharedPtr<NiceMock<application_manager::MockService> > mock_service_;
+  utils::SharedPtr<NiceMock<MockApplication> > mock_app_;
+  utils::SharedPtr<remote_control::RCAppExtension> rc_app_extention_;
+  testing::NiceMock<remote_control_test::MockRemotePluginInterface>
+      mock_module_;
+  testing::NiceMock<remote_control_test::MockResourceAllocationManager>
+      mock_allocation_manager_;
+  RemotePluginInterface::RCPluginEventDispatcher event_dispatcher_;
+  const uint32_t app_id_ = 11u;
```

Comment:
@AByzhynar, accepted - fixed in ea4a85f19c816df9c18c89177cff80e6dd121392, moved to namespace.